tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -222,7 +269,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 options: {
                     moduleSource: {
                         func: "fluid.tests.tooltipModuleSource",
-                        args: "{pager}"
+                        args: ["{pager}", [2, 3, 4]]
```

Comment:
@jobara since this parameter has been defined in the function with a meaningful name, do you think that it's required to move this to another external variable. I mean this is just a test data passed to the function. So, I think it would be complected if we separate this parameter in all the time we use this.